tests/test_flexible_unet.py

```diff
@@ -381,14 +298,14 @@ def test_shape(self, input_param, input_shape, expected_shape):
         self.assertEqual(result.shape, expected_shape)
 
     @parameterized.expand(CASES_PRETRAIN)
-    def test_pretrain(self, input_param, efficient_input_param, weight_list):
+    def test_pretrain(self, flexunet_input_param, feature_extractor_class, feature_extractor_input_param, weight_list):
         device = "cuda" if torch.cuda.is_available() else "cpu"
 
         with skip_if_downloading_fails():
-            net = FlexibleUNet(**input_param).to(device)
+            net = FlexibleUNet(**flexunet_input_param).to(device)
 
         with skip_if_downloading_fails():
-            eff_net = EfficientNetBNFeatures(**efficient_input_param).to(device)
+            eff_net = feature_extractor_class(**feature_extractor_input_param).to(device)
```

Comment:
Could you please rename the variable `eff_net` to a more general one? It means  efficient net here. And since you are adding more backbones, it should be a more general one like `feature_extractor_net`.

Thanks,
Bin

Reply:
Thank you for your feedback. I've made the necessary adjustments and renamed the variable to `feature_extractor_net `to better reflect its general purpose.

Best regards,
Konstantin